src/sinks/postgres/integration_tests.rs

```diff
@@ -0,0 +1,111 @@
+use crate::{
+    config::{SinkConfig, SinkContext},
+    sinks::{
+        postgres::PostgresConfig,
+        util::{test::load_sink, UriSerde},
+    },
+    test_util::{components::run_and_assert_sink_compliance, random_string, trace_init},
+};
+use futures::stream;
+use serde::{Deserialize, Serialize};
+use sqlx::{Connection, FromRow, PgConnection};
+use std::future::ready;
+use vector_lib::event::{BatchNotifier, BatchStatus, BatchStatusReceiver, Event, LogEvent};
+
+fn pg_host() -> String {
```

Comment:
Copied those utility functions from the `posgres_metrics` source integration tests

Reply:
Let's do some small refactoring here. 
* The folder `src/test_util` seems like a good location to put these utils.
* You can use `#[cfg(any(feature = "postgresql_metrics-integration-tests", feature = "postgres_sink-integration-tests"))]` in `src/test_util/mod.rs`.